### ISSUE TITLE
Modify NU branding logos and text on DH

### ISSUE THREAD

**Issue by kaars** [galterlibrary/digital-repository#595]
This is a request from the manager of content development at NUCATS, Clare McFadden. She asks that we try to replicate the heading they have on the NUCATS site (http://nucats.northwestern.edu/) on DH. Specifically:

1) Replace the NM/NUCATS logo with the FSM logo seen on the NUCATS site.
2) On DH change “Northwestern University Feinberg School of Medicine” to “Northwestern University Clinical and Translational Sciences Institute”.

**Comment by kristiholmes** [comment]
hi - I don't think that DH should be branded with NUCATS.  Please wait to make this change.

**Comment by kristiholmes** [comment]
I'd prefer that DH resemble our website as much as possible. Matt, maybe the two of us can touch base about this so that I can better understand the request.
Thanks!

**Comment by kaars** [comment]
Ok, sounds good. We'll hold off on the changes.

**Comment by kaars** [comment]
I talked to Kristi and we've modified the rebranding as follows:

1) Replace the NM/NUCATS logo on DH with the FSM logo seen on the NUCATS website (same as the one on the Galter website).

2) On DH remove the text “Northwestern University Feinberg School of Medicine” from the purple bar and have "DigitalHub" be the only text.

There should be no mention of “Northwestern University Clinical and Translational Sciences Institute” (NUCATS) in either text or images.
